src/config/routes.ts

```diff
@@ -2,6 +2,8 @@ import { RouteComponentProps } from '@reach/router';
 
 import LandingPage from 'src/pages/LandingPage';
 import CreateEventPage from 'src/pages/CreateEventPage';
+import UpdateEventPage from 'src/pages/UpdateEventPage';
```

Comment:
I don't see this component anywhere; did you forgot to include it? 😆

Reply:
Yep - a result of little sleep and long nights 😆